doc/design/registries/whitelist-registry.md

```diff
@@ -0,0 +1,151 @@
+# Whitelist Registries
+
+vSphere Integrated Containers 1.2 (VIC) added the ability to whitelist registry access in an installed VCH.  When a registry is added to the VCH at installed time, the VCH goes into 'whitelist' mode.  From this point on, the VCH will only allow access to registries in its list of whitelisted registries.  In this mode, users will not be able to access public registries, such as docker hub.
+
+## Specifying Whitelist Registries at Installation
+
+Whitelist registries can be declared during a VCH installation with *vic-machine* parameter, --whitelist-registry.  Two other vic-machine parameters affect whitelist registries, --registry-ca and --insecure-registry.  When whitelist registries are declared during installation, the latter two parameters acts as modifiers to the whitelist registries.
```

Comment:
Unless we want to call them "Whitelist registries".. I'm seeing that peppered through the doc, not sure if that's our intended nomenclature, ignore if so!